README.md

```diff
@@ -222,7 +222,9 @@ flask admin set-roles-to-users --filepath <path> --roles <role1>,<role2>
 
 ## Deployment
 
-`main` branch is continuously deployed to the AWS Test environment, deployments to the Dev and Prod environments are triggered by a [manual Github Actions workflow](https://github.com/communitiesuk/funding-service-design-post-award-data-store/actions/workflows/deploy.yml).
+`main` branch is continuously deployed to the AWS Test environment and requires a manual approval to be promoted to Prod. 
+
+Ad-hoc branch deployments to the Dev environment can be triggered by a [manual Github Actions workflow](https://github.com/communitiesuk/funding-service-design-post-award-data-store/actions/workflows/deploy_combined_service.yml).
```

Comment:
We can technically still manually deploy to `test` as well as dev with that workflow, don't know if we want to mention that here

Reply:
@MarcUsher Yes, good point, although an edge case I have added in 8838ef8